components/image/index.html

```diff
@@ -59,12 +59,37 @@ <h2>Responsive Image</h2>
       <p class="content-desc">Use <code>img-resp</code> class to make your images fit in the parent element</p>
       <h3>Examples</h3>
       <div class="example-box">
-        <img class="img-resp" src="../../images/avatars/profile3.jpg"></img>
+        <img class="img-resp" src="../../images/sample/sampleImg.jpg"></img>
       </div>
+      <iframe
+        src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=24px&ph=23px&ln=false&fl=1&fm=Hack&fs=13px&lh=172%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522img-resp%2522%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F400%252F400%2522%253E%253C%252Fimg%253E%250A"
+        style="width: 30rem;height: 167px;" sandbox="allow-scripts allow-same-origin">
```

Comment:
You could move the iframe style to one of the common stylesheets keeping the html clean and avoiding inline styles.

Reply:
I have done this already but the problem is height of every iframe is different for every code and here the width cant be 100% it looks bad so I did this.